### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by niels9001**
Updating the readme by removing redundant and outdated information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,120 +1,93 @@
...
+  <a href="https://aka.ms/winui-releasenotes">Release notes</a>
+</h3>
 
-WinUI is a user interface layer that contains modern controls and styles for building Windows apps. </span> As the native UI layer in Windows it embodies <a href="https://www.microsoft.com/design/fluent/#/">Fluent Design</a>, giving each Windows app the polished feel that customers expect.
+WinUI is a user interface layer that contains modern controls and styles for building Windows apps. The current generation is WinUI 3, which ships as part of the [Windows App SDK](https://aka.ms/winappsdk).
 
-WinUI 2 is a library of controls that provides official native Microsoft UI controls and features for Windows [UWP apps](https://docs.microsoft.com/windows/uwp/index). WinUI 2 can be used in any Windows 10 UWP XAML app, or in a Xamarin.Forms app running on Windows 10 using [native view embedding](https://docs.microsoft.com/xamarin/xamarin-forms/platform/native-views).
+- **Modern UI:** WinUI embodies Fluent Design to enable intuitive, accessible, and powerful experiences and the latest user interface patterns.
+**Developers in control:** Use .NET with C# or C++ to write apps that run great on x86, x64 and ARM.
```
Comment (codendone) [inline README.md:25]: Missing '- ' at the start of the line so this is its own bullet?

File: README.md [hunk 2]
```diff
@@ -1,120 +1,93 @@
...
 
-The WinUI community call is your monthly opportunity to learn about native UX development for Windows with WinUI.
+## 📋 Getting started with WinUI
 
-In these calls we’ll discuss the WinUI [roadmap](docs/roadmap.md), our status and your feedback.
+For WinUI, your app's users must be on Windows 10 1809 - Build 17763 or newer (including Windows Insider Previews).
 
-You can watch them online here on YouTube at the [Windows Developer channel](https://www.youtube.com/channel/UCzLbHrU7U3cUDNQWWAqjceA).
+The full documentation of WinUI can be found on [Microsoft Learn](https://learn.microsoft.com/en-us/windows/apps/desktop/):
+- [Get started with WinUI](https://learn.microsoft.com/en-us/windows/apps/get-started/start-here)
```
Comment (codendone) [inline README.md:40]: I think the learn.microsoft.com links should all exclude the "en-us/" part of the path, so they will choose the developer's default UI language.

**Review by codendone — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,120 +1,93 @@
...
-| [Microsoft.UI.Xaml.Core.Direct](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct) <br /> Low-level APIs for middleware components | | [![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.Core.Direct.svg)](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct/absoluteLatest) | [2.0 prerelease](https://docs.microsoft.com/uwp/api/microsoft.ui.xaml.core.direct) |
+</br>
 
-You can also build a WinUI package yourself from source. See [Contributing to the Windows UI Library](CONTRIBUTING.md) for more information on building and contributing to WinUI.
+## 🛣️ Roadmap
 
-## Documentation
+For info on the WinUI release schedule and high level plans please see the [WinUI Roadmap](https://aka.ms/winappsdk/plans).
```
Comment (eduardobragaxz) [inline README.md:76]: Shouldn't this link to the roadmap document?

File: README.md [hunk 4]
```diff
@@ -1,120 +1,93 @@
...
+  </a>
+</p>
 
-[![Follow WinUI on Twitter](https://img.shields.io/twitter/follow/windowsui.svg?label=Follow%20WinUI%20on%20Twitter&style=social)](https://twitter.com/intent/follow?screen_name=windowsui)
+<h3 align="center">
+  <a href="https://aka.ms/windev">About WinUI</a>
+  <span> · </span>
+  <a href="https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.5-stable">Source code</a>
+  <span> · </span>
+  <a href="https://learn.microsoft.com/en-us/windows/apps/desktop/">Documentation</a>
```
Comment (codendone) [inline README.md:17]: One more "en-us/" here to remove.

Comment (niels9001) [reply README.md]: Fixed!

File: README.md (hunk 3 above)
Comment (niels9001) [reply README.md:76]: @eduardobragaxz  Can you point me to that?

Comment (codendone) [reply README.md:76]: The aka.ms/winappsdk/plans link now points to the Roadmap. (We meant to update that sooner.) So I think sticking with this link should be good.

**Review by codendone — APPROVED** [review]

**Comment by Jay-o-Way** [comment]
@niels9001 why did you choose to use HTML in a MD file? Not really "in spirit" with MD.

**Pull request merged**